package.json

```diff
@@ -9,6 +9,7 @@
     "react": "15.3.2",
     "react-native": "0.34.0",
     "react-native-button": "^1.7.0",
+    "react-native-camera": "git+https://github.com/lwansbrough/react-native-camera.git",
```

Comment:
Is there no way to version lock? It seems that, depends on when you run `npm install`, you might get a different version of the package